trueusd/README.md

```diff
@@ -1,11 +1,19 @@
 # Chainlink External Adapter for trueusd
 
+## Total Supply API
+
+https://core-api.real-time-attest.trustexplorer.io/trusttoken/TrueUSD
+
 ## Input Params
 
-_No params_
+- `field`: Required, the data field to return. Must be in camel-case style.
```

Comment:
Based on the feedback from @tyrion70, we should probably make this optional and default field to `totalToken`. Is that right?